addon/helpers/choice.js

```diff
@@ -9,6 +9,11 @@ export default MicroState.extend({
       return Type.create(values, options);
     },
 
+    set(choice, values, options) {
```

Comment:
To be clear, what I'm advocating is to just not have a custom `set` action on choice.